docs/install/linux_source.rst

```diff
@@ -25,6 +25,7 @@ PyNE has the following dependencies:
 Optional Depenendencies:
    #. `MOAB <https://press3.mcs.anl.gov/sigma/moab-library>`_
    #. `PyTAPS <https://pythonhosted.org/PyTAPS/index.html>`_
```

Comment:
PyTAPS needs removed here and DAGMC needs to be added (we should probably note that DAGMC requires MOAB?)

Reply:
The link that I provided for DAGMC takes you to the page that encompasses both the link to the dependencies and the link to the installation instructions, which I think should do the trick. If I'm not mistaken, we have other dependencies that rely on each other in this way and we don't specify for those.

Reply:
ya true- probably not necessary to note here then